pkg/minikube/service/service.go

```diff
@@ -196,6 +197,13 @@ func printURLsForService(c typed_core.CoreV1Interface, ip, service, namespace st
 	urls := []string{}
 	portNames := []string{}
 	for _, port := range svc.Spec.Ports {
+
+		if port.Name != "" {
+			m[port.TargetPort.IntVal] = port.Name
```

Comment:
I don't think the port name is useful by itself.  I'd rather change to print something like `name/port` if name is available, if not only `port`.

Reply:
I agree ! if needed feel free to change the profile column name,

Reply:
Thanks for the review. I have pushed a change to address your feedback. I have added before/after images to issue description. The column name does need to be changed in my opinion. But I am not sure what would be the right name for the column.